packages/wast-parser/test/fixtures/number-literals/f32/negative-exponential/expected.json

```diff
@@ -0,0 +1,27 @@
+{
+  "type": "Program",
+  "body": [
+    {
+      "type": "Func",
+      "name": {
+        "type": "Identifier",
+        "value": "func_1"
+      },
+      "params": [],
+      "result": [],
+      "body": [
+        {
+          "type": "Instr",
+          "id": "const",
+          "object": "f32",
+          "args": [
+            {
+              "type": "FloatLiteral",
```

Comment:
I noticed something when I was working on the `wast-printer`, we don't preserve the original notation. For example: `0x01` becomes `1` because we store it as a number in the AST. I would propose introducing a `raw` fields that captures the raw string, we can then use this when printing the AST.

What do you think? We can do it in another PR since it impact some other places.

Reply:
Yep it makes tests more readable and definitely a good idea for the printer :+1: I open an issue for it.